gapis/service/service.proto

```diff
@@ -397,6 +397,9 @@ message ErrDataUnavailable {
   stringtable.Msg reason = 1;
   // If true, then making the same request at a later time may result in data.
   bool transient = 2;
+  // If true, draw call specific data was requested for a command range not
+  // containing any draw calls.
+  bool not_a_draw_call = 3;
```

Comment:
Then I think we're doing this wrong. `ErrDataUnavailable` **is** the current machine readable info/code. Do you really want to duplicate the stringtable with another set of localized messages in the client?

If you *really* want to do this, then I suggest we make the stringtable message types (`ERR_NOT_A_DRAW_CALL`) part of a proto enum that's included in the service interface.
I am opposed to this this, and I'm exceptionally unhappy with adding fields here. It sets a precedent that goes against the design of the errors and stringtables, which requires a major rethink of the system.